tests/features/rhpam/rhpam-process-migration.feature

```diff
@@ -1,11 +1,61 @@
 @rhpam-7/rhpam-process-migration-rhel8
 Feature: RHPAM Process Migration tests
 
-  Scenario: Test REST API is available and valid
+  Scenario: verify if all labels are correctly set on kogito-builder image
```

Comment:
```suggestion
  Scenario: Verify if all labels are correctly set on rhpam-process-migration-rhel8 image
```

Reply:
nice catch LOL, copy and paste issue :)

Reply:
Is still the old version, not the Jakub' suggestion

Reply:
updated, thanks.